sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -271,7 +271,12 @@ class MPTReporter implements Reporter {
       this.testResultBatch.add(testResultObject);
       // Store test attachments in array
       const testAttachments: string[] = [];
+      const regionAndSessionIdAttachment: any[] = [];
```

Comment:
let's make this generic to handle any type of attachments. We should be compliant with any other service which provides cloud browsers.